Your_first_ASDF_converter.ipynb

```diff
@@ -0,0 +1,554 @@
+    "leave it be."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "88e1f649",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mkdir ~/converter_tutorial"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3b1e163f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cd ~/converter_tutorial"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "06a42933",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Record current directory for restarts\n",
+    "import os\n",
+    "curdir = os.getcwd()\n",
+    "print(curdir)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0ae4a184",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mkdir mfconverter"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "46d22a46",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cd mfconverter"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a980c3f9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mkdir src"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8dbb76d3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mkdir src/mfconverter"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ede348bd",
+   "metadata": {},
+   "source": [
+    "And now we will create a module that has a very, very simple photo ID class and add a package `__init__.py`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f045f111",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile src/mfconverter/photo_id.py\n",
+    "class PhotoID:\n",
+    "    \"Holds Phot ID information\"\n",
+    "\n",
+    "    def __init__(self, last_name, first_name, image):\n",
+    "        \"expects a monochromatic numpy array for image\"\n",
+    "        self.last_name = last_name\n",
+    "        self.first_name = first_name\n",
+    "        self.photo = image\n",
+    "    \n",
+    "    def name(self):\n",
+    "        return self.last_name + ', ' + self.first_name"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ad205d2e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile src/__init__.py\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "09ee1385",
+   "metadata": {},
+   "source": [
+    "Next we create the file that contains the converter code"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "aa345e79",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "%%writefile src/mfconverter/converter.py\n",
+    "from asdf.extension import Converter\n",
+    "from .photo_id import PhotoID\n",
+    "\n",
+    "class PhotoIDConverter(Converter):\n",
+    "    tags = [\"asdf://stsci.edu/example-project/photo_id-*\"]\n",
```

Comment:
In the asdf documentation we're recommending `example-project/tags/photo_id-*` as a convention for tag path.  We should either change the docs or this example so that they match.

Reply:
(https://asdf.readthedocs.io/en/latest/asdf/extending/uris.html#tags)

Reply:
Changing to be consistent.